Fix D3(BJ)-ATM calculations in DFTB+ (#890)
dftbplus/dftbplus: CHANGELOG.rst

- Updated version of dftd3-lib fixes a numerical typo in the Ba parameters

> - D3(BJ)-ATM calculator did not pass exponent for ATM zero damping function

Reviewer: ```suggestion
- D3(BJ)-ATM calculator was not being passed the exponent  for ATM zero damping calculations
```
